src/sentry/sentry_metrics/indexer/id_generator.py

```diff
@@ -0,0 +1,50 @@
+import random
+import time
+
+_VERSION_BITS = 4
+_TS_BITS = 32
+_RANDOM_BITS = 28
+_TOTAL_BITS = _VERSION_BITS + _TS_BITS + _RANDOM_BITS
+assert _TOTAL_BITS == 64
+
+_VERSION = 2
```

Comment:
Version 1 is implicit here 
https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/indexer/strings.py#L28

I think we should define the used versions in the same place. But you can do it later.